specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -278,21 +279,49 @@ function MergeButton<SCHEMA extends AnySchema>({
     );
   }, [mergeResource]);
 
+  const [warningDialog,_,handleCloseWarningDialog, toggleWarningDialog] = useBooleanState(false)
+  const [isAccepted, setIsAccepted, __, ___] = useBooleanState(false)
+
   return (
     <>
       {!saveBlocked ? (
-        <Submit.Blue form={formId}>{treeText.merge()}</Submit.Blue>
+        <>
+          {isAccepted ?
+          <Submit.Green form={formId}>{treeText.merge()}</Submit.Green> 
+          : 
+          <Button.Info onClick={toggleWarningDialog}>{commonText.proceed()}</Button.Info>
+          }
+        </>
       ) : (
-        <Submit.Red className="cursor-not-allowed">
+        <Button.Danger onClick={undefined} className="cursor-not-allowed">
```

Comment:
was the change from `Submit` to `Button` intentional? should be ok, just want to double check

Reply:
yes but can revert is it's better to have submit.red